sdk/monitor/azure-monitor-opentelemetry-exporter/README.md

```diff
@@ -80,7 +99,21 @@ Some of the key concepts for the Azure monitor exporter include:
 
 * [AzureMonitorLogExporter][log_reference]: This is the class that is initialized to send logging related telemetry to Azure Monitor.
 
-* [Trace][trace_concept]: Trace refers to distributed tracing. It can be thought of as a directed acyclic graph (DAG) of `Span`s, where the edges between `Span`s are defined as parent/child relationship.
+* [Metric][metric_concept]: `Metric` refers to recording raw measurements with predefined aggregation and sets of attributes for a period in time.
+
+* [Measurement][measurement]: Represents a data point recorded at a point in time.
+
+* [Instrument][instrument]: Instruments are used to report `Measurement`s.
+
+* [Meter][meter]: The `Meter` is responsible for creating `Instruments`.
+
+* [Meter Provider][meter_provider]: Provides a `Meter` for the given instrumentation library.
+
+* [Metric Reader][metric_reader]: An SDK implementation object that provides the common configurable aspects of the OpenTelemetry Metrics SDK such as collection, flushing and shutdown.
+
+* [AzureMonitorMetricExporter][metric_reference]: This is the class that is initialized to send metric related telemetry to Azure Monitor.
+
+* [Trace][trace_concept]: Trace refers to distributed tracing. A distributed trace is a set of events, triggered as a result of a single logical operation, consolidated across various components of an application.
```

Comment:
This feels vague compared to the previous span description. Could we make it more concrete? Perhaps with an example?

Reply:
Re-added previous definition mentioning the DAG.